scripts/build-vercel-functions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+VERCEL_FUNCTIONS_SOURCE="vercel-functions"
```

Comment:
Same note as above -- try to avoid indirection unless it truly helps with readability. Here I don't think it applies to most values extracted out as constants